<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordFormatBar</class>
 <widget class="QWidget" name="WordFormatBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>105</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="gbWords">
       <property name="title">
        <string>Redistribute Words</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Words per line:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbWords">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbSpace">
       <property name="title">
        <string>Fill Space</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="cbFillSpaces">
          <property name="text">
           <string>Use blanks:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="cbFillPattern">
          <property name="text">
           <string>Use pattern:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leFillPattern">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbFillBlanks">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbAlign">
       <property name="title">
        <string>Table Layout</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cbCell">
          <property name="text">
           <string>Cell width:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sbCell">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> char</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="cbFitWords">
          <property name="text">
           <string>Expand to maximum</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbFormat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gbWords</tabstop>
  <tabstop>sbWords</tabstop>
  <tabstop>cbFillSpaces</tabstop>
  <tabstop>sbFillBlanks</tabstop>
  <tabstop>cbFillPattern</tabstop>
  <tabstop>leFillPattern</tabstop>
  <tabstop>cbCell</tabstop>
  <tabstop>sbCell</tabstop>
  <tabstop>cbFitWords</tabstop>
  <tabstop>pbFormat</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbFillPattern</sender>
   <signal>clicked()</signal>
   <receiver>leFillPattern</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFillSpaces</sender>
   <signal>clicked()</signal>
   <receiver>sbFillBlanks</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
